lib/kitchen/directions/bake_images.rb

```diff
@@ -3,9 +3,9 @@
 module Kitchen
   module Directions
     module BakeImages
-      def self.v1(book:, resources:)
+      def self.v1(book_pages:, resources:)
```

Comment:
since all web directions work within pages, not the full book, i've updated web-only directions to expect a pages enum